tests/resources/naive_MDSdataset/25/index.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
could we call this dir `naive_mds_dataset`, mixing underscores and caps gives me the heebie jeebies

Reply:
removed the dataset. creating test mds dataset on the fly.